pkg/cmd/server/origin/asset.go

```diff
@@ -141,6 +142,18 @@ func (c *AssetConfig) buildAssetHandler() (http.Handler, error) {
 	return handler, nil
 }
 
+// Have to convert to arrays because go templates are limited and we need to be able to know
+// if we are on the last index for trailing commas in JSON
+func extensionPropertyArrays(extensionProperties map[string]string) ([]template.JSStr, []template.JSStr) {
+	extensionKeys := []template.JSStr{}
```

Comment:
we really shouldn't need to pre-escape these... the template should be able to do that for us. I suspect we had weird values in the strings (possibly from YAML doing odd things with quotes within quotes?)